tests/xgboost/test_xgboost_converters.py

```diff
@@ -303,6 +303,40 @@ def test_xgb_empty_tree(self):
         assert_almost_equal(xgb.predict_proba(X), res[1])
         assert_almost_equal(xgb.predict(X), res[0])
 
+    def test_xgb_best_tree_limit(self):
+
+        # Train
+        iris = load_iris()
+        X, y = iris.data, iris.target
+        X_train, X_test, y_train, y_test = train_test_split(X, y)
+        dtrain = DMatrix(X_train, label=y_train)
+        dtest = DMatrix(X_test)
+        param = {'objective': 'multi:softmax', 'num_class': 3}
+        bst_original = train_xgb(param, dtrain, 10)
+        initial_type = [('float_input', FloatTensorType([None, 4]))]
+        bst_original.save_model('model.json')
+
+        onx_loaded = convert_xgboost(bst_original, initial_types=initial_type)
```

Comment:
You should add something target_opset=TARGET_OPSET to make sure that the conversion is done with an opset onnxruntime supports.

Reply:
I should close PR #506.

Reply:
added